### PR TITLE
Fix Manual color mode

### LINKED ISSUE #213: Manage colors globally
Manage colors globally and by component

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
fix #213

### CODE REVIEW

File: src/app/services/main-form-manager/config-export-helper.ts [hunk 1]
```diff
@@ -68,13 +69,13 @@ export class ConfigExportHelper {
...
         analyticsConfigList: FormArray,
-        keysToColorList: FormArray,
+        colorService: ArlasColorGeneratorLoader,
     ): any {
         const chipssearch: ChipSearchConfig = {
             name: searchConfigGlobal.customControls.name.value,
             icon: searchConfigGlobal.customControls.unmanagedFields.icon.value
         };
-
+        console.log(colorService.keysToColors)
```
Comment (sebbousquet) [inline src/app/services/main-form-manager/config-export-helper.ts:78]: 👎  😈

File: src/app/services/main-form-manager/main-form-manager.service.ts [hunk 2]
```diff
@@ -207,6 +209,7 @@ export class MainFormManagerService {
   public doImport(config: Config, mapConfig: MapConfig) {
 
     const startingConfigControls = this.mainFormService.startingConfig.getFg().customControls;
+    console.log(config.arlas.web.colorGenerator)
```
Comment (sebbousquet) [inline src/app/services/main-form-manager/main-form-manager.service.ts:212]: 👎

File: src/app/shared/services/property-selector-form-builder/property-selector-form-builder.service.ts [hunk 3]
```diff
@@ -191,23 +191,30 @@ export class PropertySelectorFormGroup extends ConfigFormGroup {
               if (result !== undefined) {
                 this.customControls.propertyManualFg.propertyManualValuesCtrl.clear();
                 result.forEach((kc: KeywordColor) => {
+                  // TODO: release new toolkit
+                  /** after closing the dialog, save the [keyword, color] list in the Arlas color service */
+                  console.log(kc)
```
Comment (sebbousquet) [inline src/app/shared/services/property-selector-form-builder/property-selector-form-builder.service.ts:196]: 👎

Comment (sebbousquet) [reply src/app/shared/services/property-selector-form-builder/property-selector-form-builder.service.ts:196]: Two others below

File: src/app/modules/analytics-config/services/powerbar-form-builder/powerbar-form-builder.service.ts [hunk 4]
```diff
@@ -85,6 +85,8 @@ export class PowerbarConfigForm extends ConfigFormGroup {
           marker('Powerbar manage colors description'),
           () => collectionService.getTermAggregation(collection, this.customControls.dataStep.aggregationField.value)
             .then((keywords: Array<string>) => {
+              console.log(colorService)
```
[the remark this answers is not in the document]
Comment (MohamedHamouGisaia) [reply src/app/modules/analytics-config/services/powerbar-form-builder/powerbar-form-builder.service.ts]: fixed

File: src/app/modules/map-config/components/layers/layers.component.ts [hunk 5]
```diff
@@ -148,10 +147,12 @@ export class LayersComponent implements OnInit {
     }
     currentConfig.arlas.web.contributors.push(contribConfig);
     this.configService.setConfig(currentConfig);
+    console.log(this.colorService)
```
[the remark this answers is not in the document]
Comment (MohamedHamouGisaia) [reply src/app/modules/map-config/components/layers/layers.component.ts]: fixed

File: src/app/services/main-form-manager/config-export-helper.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/services/main-form-manager/config-export-helper.ts]: fixed

File: src/app/services/main-form-manager/main-form-manager.service.ts (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/app/services/main-form-manager/main-form-manager.service.ts]: fixed

File: src/app/shared/services/property-selector-form-builder/property-selector-form-builder.service.ts (hunk 3 above)
Comment (MohamedHamouGisaia) [reply src/app/shared/services/property-selector-form-builder/property-selector-form-builder.service.ts]: fixed

**Review by sebbousquet — APPROVED** [review]

**Pull request merged**